environments.json

```diff
@@ -1265,7 +1265,7 @@
       "non-standard"
     ]
   },
-  "safari51": {
```

Comment:
Do you think it'd be easier to merge yours first, and then rebase this one? or would it be easier to merge this, and rebase yours on top?